src/azure-cli/azure/cli/command_modules/appservice/static_sites.py

```diff
@@ -562,7 +562,14 @@ def reset_staticsite_api_key(cmd, name, resource_group_name=None):
                                             reset_properties_envelope=reset_envelope)
 
 
-def link_user_function(cmd, name, resource_group_name, function_resource_id, force=False):
+def link_user_function(
+    cmd,
+    name,
+    resource_group_name,
+    function_resource_id,
+    environment_name: str,
```

Comment:
Please set the default value for parameter `environment_name`, otherwise it will become a required parameter and cause breaking changes
```suggestion
    environment_name=None,
```